page_forms/basket_page.py

```diff
@@ -0,0 +1,16 @@
+from .base_page import BasePage
+from resource.locators import BasketPageLocators
+
+
+class BasketPage(BasePage):
+    def __init__(self, browser, url):
+        super().__init__(browser, url)
+
+    def basket_should_be_empty(self):
+        assert self.is_not_element_present(*BasketPageLocators.BASKET_ITEM), \
+            "There should be no items in the list"
+
+    def text_basket_is_empty_should_be_present(self):
```

Comment:
To avoid duplication of code , need to implement some useful functions in BasePage class to work with web elements: find_element, find_elements, get_element_text, click_element, enter_text, etc...

 def check_basket_alert_message(self, expected_alert):
        alert = self.get_element_text(*BasketPageLocators.ALERT)
        assert expected_alert in alert, "Text 'Basket is empty' should be present"